src/provider/rpc.ts

```diff
@@ -203,18 +204,18 @@ export class RpcProvider implements ProviderInterface {
       .then(this.responseParser.parseContractClassResponse);
   }
 
-  public async getContractVersion(
+  public async getContractSpecificities(
```

Comment:
@tabaktoni did you mean to rename also all instances of "specificities" in the PR?

Reply:
Yea

Reply:
Modified